pkg/apic/client.go

```diff
@@ -335,7 +349,7 @@ func (c *ServiceClient) checkAPIServerHealth() error {
 
 	if c.cfg.GetTeamID() == "" {
 		// Validate if team exists
-		team, err := c.getCentralTeam(c.cfg.GetTeamName())
+		team, err := c.GetCentralTeamByName(c.cfg.GetTeamName())
```

Comment:
Is this needed anymore with the addition in the agent package that was made? initEnvResources

Reply:
You are right, the checkAPIServerHealth method is left over, will remove the method as it is not used anymore

Reply:
Done